go.sum

```diff
@@ -0,0 +1,49 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Can we move these files into the `go` dir?

Reply:
we cannot move go.mod to `./go` dir because it needs to be somewhere that is a common ancestor of all the other go files in this project e.g. ./sample ./grpc

So like package.json, it goes into the project root.
In addition, go.sum gets automatically generated by the go toolset, precisely where go.mod is.
(go.sum is analogous to package-lock.json). Convension is to check both into version control